pkg/kritis/cryptolib/verifier.go

```diff
@@ -16,7 +16,18 @@ limitations under the License.
 
 package cryptolib
 
-import "errors"
+import (
+	"errors"
+	"fmt"
+)
+
+// TODO: Remove function stubs once implemented
+// For testing
+var (
+	pkixVerify                      = verifyPkix
```

Comment:
I think you are trying to alias functions so you can stub them out for testing. Instead of using global variables for this, have verifier contain a private interface(s) which it uses. In the non-test case, when you construct a verifier it should automatically set the contained interface to the real implementation, but your unit tests can reach into the internals and replace it with a stub.

Reply:
I tried following this guidance. Please take a look to make sure I followed your suggestions appropriately.